core/services/flux_monitor_test.go

```diff
@@ -347,10 +376,12 @@ func TestPollingDeviationChecker_NoDeviation_CanBeCanceled(t *testing.T) {
 		Return(big.NewInt(1), nil)
 	ethClient.On("SubscribeToLogs", mock.Anything, mock.Anything).
 		Return(fakeSubscription(), nil)
+	ethClient.On("GetLatestSubmission", mock.Anything, mock.Anything).
```

Comment:
It seems like all the tests that have `GetLatestSubmission` mocked cover the case where no run is created. Can we get a test with the positive case, or am I missing it?

Reply:
@se3000 in the first test, `TestPollingDeviationChecker_TriggerIdleTimeThreshold` jobs do get run, but the rounds are irrelevant anyway because it's triggered by the idleThreshold expiring. I added a new, pretty comprehensive test `TestPollingDeviationChecker_PollIfRoundOpen` that more explicitly tests the functionality of polling based on round eligibility.

Reply:
Awesome, that looks great. Thank you.